docs/contributing/index.md

```diff
@@ -1,310 +1,135 @@
 
-    ```shell
-    git commit -m "My descriptive commit message"
-    git push
-    ```
+To create a change log entry or news item, create a file that is placed in the root of the repository directory at `/news`.
+Its format must be `###.type`, where `###` is the referenced GitHub issue or pull request number, `.` is the literal extension delimiter, and `type` is one of the following strings.
 
-1.  Visit the GitHub `documentation` repository, and [create a pull request](https://github.com/plone/documentation/compare) against the branch `6-dev`.
-1.  Members who subscribe to the repository will receive a notification and review your request.
-1.  Request a review from other team members.
+-   `breaking` for breaking changes
+-   `bugfix` for bug fixes
+-   `documentation` for documentation
+-   `feature` for new features
+-   `internal` for internal changes
 
+A project configures the types it allows in a file `towncrier.toml` located at the root of its repository.
 
-(contributing-editing-external-package-documentation-label)=
+The content of this file must include the following.
 
-### Editing external package documentation
+-   A brief message that summarizes the changes in your contribution.
+-   An attribution to yourself, in the format of `@github_username`.
 
-If you want to edit documentation of imported external packages, the process is slightly different.
-We use `git submodules` to manage multiple repositories.
-We imported the external repositories the `plone/documentation` repository as described in {doc}`setup-build`.
+The following text is an example change log entry, placed inside {file}`/news/4569.documentation`.
 
-```{important}
-We currently use the branches `plone/documentation@6-dev`, `plone/plone.api@master`, `plone/plone.restapi@master`, and `plone/volto@master` as the main branches for developing Plone 6 Documentation.
-These branches may change as we get closer to a production release.
+```text
+Fix broken links for ReactJS.org. @stevepiercy
 ```
 
-1.  From the project root directory, sync your local `6-dev` branch with its remote.
-    You might need to resolve conflicts.
-
-    ```shell
-    git checkout 6-dev
-    git pull
-    ```
-
-1.  Change your working directory to the imported package's directory under `submodules/`.
-
-    ```shell
-    # Choose one.
-    cd submodules/plone.api
-    cd submodules/plone.restapi
-    cd submodules/volto
-    ```
-
-1.  Update the submodule, and sync your local development branch with its remote.
-    You might need to resolve conflicts.
-
-    ```shell
-    git submodule update
-
-    # for plone.api
-    git checkout master
 
-    # for plone.restapi
-    git checkout master
+(contributing-specific-contribution-policies-for-packages-label)=
 
-    # for volto
-    git checkout master
+## Specific contribution policies of packages
 
-    git pull
-    ```
+Each Plone package may have specific contribution policies and guidance.
```

Comment:
I'ts not only packages. But maybe we should indeed 'promote' Products.CMFPlone as the main representative package for the backend/classic-UI for now. Mabye call it 'meta packages' or 'main packages'.  And add a sentence like  "Main packages represent the global area's in which we develop together on the code base." .   Please reword. ;-)

Reply:
Fixed in https://github.com/plone/documentation/pull/1478/commits/e7a15204751163efe67d514dd6448478465a33d4